src/main/java/mate/service/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package mate.service.impl;
+
+import mate.dao.DriverDao;
+import mate.exception.AuthenticationException;
+import mate.lib.Inject;
+import mate.lib.Service;
+import mate.model.Driver;
+import mate.service.AuthenticationService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverDao driverDao;
```

Comment:
Let's use service here

Reply:
Why we can't use dao here?

Reply:
Our service isn't doing any extra logic with driver for us to call it